src/components/Button/Button.tsx

```diff
@@ -72,7 +75,7 @@ export function Button<T = {}>(props: IButton<T>): React.ReactElement | null {
   const IconRight = iconRight;
   const withIcon = !!iconLeft || !!iconRight;
   const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
-    if (!disabled && onClick) {
+    if (!disabled && !loading && onClick) {
```

Comment:
лучше уже вынести условие в отдельную переменную с понятным названием, например

```suggestion
    const isInteractive = !disabled && !loading && onClick;
    
    if (!disabled && !loading && onClick) {
```

Reply:
Да, так можно сделать, но TS не понимает понимает такой логики и ругается на возможный undefined для onClick

Reply:
<img width="886" alt="1" src="https://user-images.githubusercontent.com/45596310/80500709-2f8b6700-8977-11ea-8ddd-5f13bbe0947a.png">

Reply:
ладно тогда